sample/get-bucket.c

```diff
@@ -15,16 +16,14 @@ int main(void) {
     uint64_t i;
 
     // Setup client credentials and then the actual client itself.
-    ds3_creds * creds = ds3_create_creds("cnlhbg==","ZIjGDQAs");
-    ds3_client * client = ds3_create_client("http://192.168.56.101:8080", creds);
+    ds3_creds * creds = ds3_create_creds(DS3_ACCESS_KEY, DS3_SECRET_KEY);
```

Comment:
Instead of having this all defined in a header file, just use the `ds3_create_client_from_env` call instead.